### PR TITLE
GH#11 --skip-config feature added

### PR DESCRIPTION
**Pull request by Sidsector9**
--skip-config will avoid addition of multisite constants to wp-config.php file.

### CODE REVIEW

**Comment by schlessera** [comment]
Fixes #11

File: src/Core_Command.php [hunk 1]
```diff
@@ -636,7 +639,9 @@ private function _multisite_convert( $assoc_args ) {
 EOT;
 
 			$wp_config_path = Utils\locate_wp_config();
-			if ( is_writable( $wp_config_path ) && self::modify_wp_config( $ms_config ) ) {
+			if ( true === \WP_CLI\Utils\get_flag_value( $assoc_args, 'skip-config' ) ) {
+				WP_CLI::log( "Addition of multisite constants to 'wp-config.php' skipped. You need to add them manually:" . PHP_EOL );
```
Comment (schlessera) [inline src/Core_Command.php]: Typo: The message should end with a period (`.`), not a colon (`:`).

**Comment by schlessera** [comment]
@Sidsector9 Thanks for the pull request!
Can you please add a functional test to assert that the flag works as intended?
Here's more information about writing tests: https://make.wordpress.org/cli/handbook/pull-requests/#running-and-writing-tests

File: src/Core_Command.php (hunk 1 above)
Comment (danielbachhuber) [inline src/Core_Command.php]: Or, we could include the constants similar to below ` You need to add them manually:" . PHP_EOL . $ms_config`

Comment (Sidsector9) [inline src/Core_Command.php]: @danielbachhuber I think it'll be a good idea to let the user have a copy of the multisite constants, this was they can use it later if they change their mind? What do you say?
CC: @schlessera

**Comment by Sidsector9** [comment]
@schlessera I'll be working on the functional tests soon.

File: src/Core_Command.php (hunk 1 above)
Comment (danielbachhuber) [inline src/Core_Command.php]: >  I think it'll be a good idea to let the user have a copy of the multisite constants, this was they can use it later if they change their mind?

I agree, let's continue to print the constants.

**Comment by Sidsector9** [comment]
@schlessera Functional test added.

**Pull request merged**